app/jsx/components/UnitSeriesComp.jsx

```diff
@@ -8,7 +8,7 @@ import faker from 'faker/locale/en'
 
 class UnitSeriesComp extends React.Component {
   componentDidMount() {
-    $('.c-pub__heading, .c-pub__abstract').dotdotdot({watch: 'window'
+    $('.c-pub__heading, .c-pub__abstract').dotdotdot({watch: 'window', tolerance: 3,
```

Comment:
What is the tolerance parameter?

Reply:
It's described here:
https://dotdotdot.frebsite.nl/#options

I found out about it through a reference on this page:
https://stackoverflow.com/questions/36258246/intermittent-truncation-using-jquery-dotdotdot

It essentially lets the dotdotdot plugin "play nice" with MathJax.